docs/Reference/Hooks.md

```diff
@@ -412,6 +414,25 @@ fastify.addHook('onReady', async function () {
   await loadCacheFromDatabase()
 })
 ```
+### onListen
+
+Triggered after the server starts listening for requests.
+It is useful if you have code that needs to be executed directly
```

Comment:
The text is repetetive. 

@Fdawgs can you have a look over the wording, please?